src/lib/mdns/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -370,14 +370,14 @@ CHIP_ERROR AdvertiserMinMdns::Advertise(const CommissionAdvertisingParameters &
     {
         return CHIP_ERROR_NO_MEMORY;
     }
-    const char * serviceType =
-        params.GetCommissionAdvertiseMode() == CommssionAdvertiseMode::kCommissionableNode ? "_chipc" : "_chipd";
-
-    FullQName operationalServiceName = AllocateQName(serviceType, "_udp", "local");
-    FullQName operationalServerName  = AllocateQName(nameBuffer, serviceType, "_udp", "local");
+    const char * serviceType = params.GetCommissionAdvertiseMode() == CommssionAdvertiseMode::kCommissionableNode
+        ? kCommissionableServiceName
+        : kCommissionerServiceName;
+    FullQName operationalServiceName = AllocateQName(serviceType, kCommissionProtocol, kLocalDomain);
```

Comment:
`operationalServiceName` is misleading. Was it copied from the operational advertising code? Could you fix it by the way?